src/views/citation_editable.tsx

```diff
@@ -90,6 +94,37 @@ export class CitationEditableView extends CitationView<
       this.props.popper.update()
     }
 
+    const findPosition = (doc: ManuscriptNode, id: string) => {
+      let nodePos: number | undefined = undefined
+
+      doc.descendants((node, pos) => {
+        if (node.attrs.id === id) {
+          nodePos = pos
+        }
+      })
+
+      return nodePos
+    }
+
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    const handleSave = async (data: any) => {
```

Comment:
also Partial Model

Reply:
Updated to be `Partial<BibliographyItem>`.